server/server.go

```diff
@@ -23,24 +24,33 @@ type Server struct {
 	CertFile       string // Server certificate PEM file
 	KeyFile        string // Server key PEM file
 	ClientCertFile string // Trusted client certificate PEM file for client authentication
+	Insecure       bool   // run without TLS
 }
 
 // Start initializes a server to respond to HTTPS/TLS network requests.
 func (s *Server) Start(ctx context.Context) error {
-	tlsOptions := tlsconfig.Options{
-		CAFile:             s.CAFile,
-		CertFile:           s.CertFile,
-		KeyFile:            s.KeyFile,
-		ExclusiveRootPools: true,
+	var tlsConfig *tls.Config
+	if s.Insecure {
+		tlsConfig = nil
+		logrus.Warnln("RUNNING IN INSECURE MODE")
+		logrus.Warnln("RUNNING IN INSECURE MODE")
```

Comment:
Remove duplicate log lines

Reply:
i would prefer to leave it in, as i dont want an end user to accidentally miss that they are running in insecure mode.